db/migrate/20220708015711_add_columns_to_plant.rb

```diff
@@ -0,0 +1,10 @@
+class AddColumnsToPlant < ActiveRecord::Migration[7.0]
+  def change
+    add_column :plants, :img, :string
```

Comment:
we don't need `:img` as a `string`.
attachments will be managed by `rails active_storage:install` and declaring in a model file: `has_one_attached :image` or `has_many_attachments :images`

Reply:
also within this PR you can run
rails `rails active_storage:install` on your machine, it'll create additional migration and then you can add in a model this: `has_one_attached :img`